packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -309,18 +309,21 @@ export class DocumentDeltaConnection
         this.submitCore("submitSignal", [message]);
     }
 
+    protected disposeSocket(error: IAnyDriverError) {
+        this.disposeCore(error);
+    }
```

Comment:
perhaps add the comment you mentioned when we chatted.  At the driver-base layer it does not matter disposeSocket vs disposeCore but at the odsp-driver, it is actually where the socket gets  closed and disposeCore is called.